docs/contracts/v2/guides/01-local-environment.md

```diff
@@ -26,11 +26,16 @@ In addition, familiarity with [Ethereum](https://ethereum.org/) and [Solidity](h
 We put together a template repository that you can use to get started quickly. This repository features a basic project
 structure, pre-configured Sablier imports, and a selection of sample contracts and tests.
 
+:::tip
+Make sure you are using the latest version of Foundry by running `foundryup` before executing the following
+commands.
+:::
+
```

Comment:
Needs to be formatted like this due to a Prettier bug (notice the extra end lines):

```suggestion
:::tip

Make sure you are using the latest version of Foundry by running `foundryup` before executing the following
commands.

:::
```